continuationsPlugin/src/test/scala/continuations/ContinuationsPluginSuite.scala

```diff
@@ -504,14 +504,16 @@ class ContinuationsPluginSuite extends FunSuite, CompilerFixtures {
            |      {
            |        val x: Int = 5
            |        println("HI")
-           |        val continuation1: continuations.Continuation[Int] = completion
-           |        val safeContinuation: continuations.SafeContinuation[Int] = 
-           |          new continuations.SafeContinuation[Int](continuations.intrinsics.IntrinsicsJvm$package.intercepted[Int](continuation1)(), 
-           |            continuations.Continuation.State.Undecided
-           |          )
-           |        val suspendContinuation: Int = 0
-           |        safeContinuation.resume(Right.apply[Nothing, Int](1))
-           |        safeContinuation.getOrThrow()
+           |        {
```

Comment:
how come we only need to change this test and not the others before it?

Reply:
Good question! Because there was no other rows before the continuation, so the body of the function was the whole continuation block.